src/lib/Tab/LocationView/AuthorsTab.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\Tab\LocationView;
+
+use ArrayObject;
+use Ibexa\AdminUi\Specification\UserExists;
+use Ibexa\AdminUi\UI\Dataset\DatasetFactory;
+use Ibexa\Contracts\AdminUi\Tab\AbstractEventDispatchingTab;
+use Ibexa\Contracts\AdminUi\Tab\OrderedTabInterface;
+use Ibexa\Contracts\Core\Repository\SectionService;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\Contracts\Core\Repository\Values\Content\ContentInfo;
+use Ibexa\Contracts\Core\Repository\Values\Content\VersionInfo;
+use JMS\TranslationBundle\Annotation\Desc;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Twig\Environment;
+
+class AuthorsTab extends AbstractEventDispatchingTab implements OrderedTabInterface
+{
+    public const URI_FRAGMENT = 'ibexa-tab-location-view-authors';
+
+    protected UserService $userService;
+
+    protected SectionService $sectionService;
+
+    protected DatasetFactory $datasetFactory;
+
+    public function __construct(
+        Environment $twig,
+        TranslatorInterface $translator,
+        SectionService $sectionService,
+        UserService $userService,
+        DatasetFactory $datasetFactory,
+        EventDispatcherInterface $eventDispatcher
+    ) {
+        parent::__construct($twig, $translator, $eventDispatcher);
+
+        $this->sectionService = $sectionService;
+        $this->userService = $userService;
+        $this->datasetFactory = $datasetFactory;
+    }
+
+    public function getIdentifier(): string
+    {
+        return 'authors';
+    }
+
+    public function getName(): string
+    {
+        /** @Desc("Authors") */
+        return $this->translator->trans('tab.name.authors', [], 'ibexa_locationview');
+    }
+
+    /**
+     * @return int
+     */
+    public function getOrder(): int
+    {
+        return 200;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getTemplate(): string
+    {
+        return '@ibexadesign/content/tab/authors.html.twig';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getTemplateParameters(array $contextParameters = []): array
+    {
+        /** @var \Ibexa\Contracts\Core\Repository\Values\Content\Content $content */
+        $content = $contextParameters['content'];
+
+        $versionInfo = $content->getVersionInfo();
+        $contentInfo = $versionInfo->getContentInfo();
+
+        $viewParameters = new ArrayObject([
```

Comment:
Is there any particular reason you need `ArrayObject` instead of simple `array`? Looks counterintuitive to me given you call `$viewParameters->getArrayCopy()` in the end.

Reply:
`ArrayObject` only makes sense in context of a `json_encode` operation, to ensure that it is properly encoded as an empty array in certain cases. This is no such case.

Reply:
This is still there @kisztof.